packages/create-llama/helpers/typescript.ts

```diff
@@ -120,26 +120,19 @@ export const installTSTemplate = async ({
     await Promise.all([fs.rm(nextConfigO11yPath), fs.rm(nextConfigAppPath)]);
   }
 
-  if (observability) {
-    const routeO11yPath = path.join(
-      root,
-      "app",
-      "api",
-      "chat",
-      "route.o11y.ts",
-    );
-    const routePath = path.join(root, "app", "api", "chat", "route.ts");
-    await fs.rm(routePath);
-    await fs.rename(routeO11yPath, routePath);
-  } else {
-    const routeO11yPath = path.join(
-      root,
-      "app",
-      "api",
-      "chat",
-      "route.o11y.ts",
+  if (observability === "OpenTelemetry") {
+    const openLLMetryPath = path.join(
+      templatesDir,
+      "components",
+      "observability",
+      "typescript",
+      "openllmetry",
+      "observability.ts",
```

Comment:
Sorry, I didn't explain that 100%:
To be aligned with the engine code, please copy the whole folder (variable `observability` contains the folder name - use `index.ts` as filenames)

```suggestion
  if (observability !== "none") {
    const openLLMetryPath = path.join(
      templatesDir,
      "components",
      "observability",
      "typescript",
         const openLLMetryFolder = path.join(
      templatesDir,
      "components",
      "observability",
      "typescript",
      observability
```

Reply:
Done!